arelle/plugin/validate/ESEF/__init__.py

```diff
@@ -364,6 +360,12 @@ def modelTestcaseVariationReportPackageIxdsOptions(
 __pluginInfo__ = {
     # Do not use _( ) in pluginInfo itself (it is applied later, after loading
     "name": "Validate ESMA ESEF",
+    "aliases": [
+        # Aliases for when ESEF validation was handled by multiple plugins.
+        "Validate ESMA ESEF-2022",
+        "validate/ESEF_2022",
+        "validate\ESEF_2022",
```

Comment:
Fine to leave this, but aliases (and the search terms compared against them) do get normalized to replace `\` with `/` so these will evaluate to the same alias.